templates/includes/sidebar.html

```diff
@@ -0,0 +1,8 @@
+{% block sidebar %}
```

Comment:
You don't need "block "tag here, cause you include this template in the base.html

Reply:
But, what if I want to change it on some pages? I guess I have to use it. Moreover, in the next tasks in mentor's code in file 'base.html' this block does exist